src/main/java/Tasks.java

```diff
@@ -1,8 +1,8 @@
 import java.util.ArrayList;
 
 public class Tasks {
-    boolean isCompleted;
-    String name;
+    public boolean isCompleted;
```

Comment:
I like how descriptive the variables are named as. However perhaps it will be better if these individual class attributes can be kept private or protected instead?